myapp/segmentsconfigapp/serializers.py

```diff
@@ -0,0 +1,30 @@
+from segmentsconfigapp.models import SegmentsConfig
+
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+from segmentsconfigapp.parser import ParserSegmentConfig
+
+#Use the parser
+def parse_rules_validator(value):
```

Comment:
it might be cleaner if you have only serializers inside a serializers.py, and create a new file validators.py with only validators